### PR TITLE
listing page added

### PR DESCRIPTION
**Pull request by pariyar07**

### CODE REVIEW

File: pages/listing.html [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+                            <label for="1star">1 Star & above</label>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </aside>
+        <section id="main-doc">
+            <div class="listing-card-section">
+                <div class="card">
+                    <img class="card-img" src="../assets/images/raptor-img.jpeg" alt="" />
```
Comment (Aditi0505) [inline pages/listing.html:114]: Suggestion: Add alt description for accessibility.

File: pages/listing.html [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+                            <p class="discount-percentage">(50% OFF)</p>
+                        </div>
+                    </div>
+                    <div class="bottom-btn cart">
+                        <button class="btn default add-cart">Add to Cart</button>
+                    </div>
+                </div>
+
+                <div class="card">
+                    <img class="card-img" src="../assets/images/raptor-img.jpeg" alt="" />
```
Comment (Aditi0505) [inline pages/listing.html:135]: Suggestion: Add alt description for accessibility.

File: pages/listing.html [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
```
Comment (Aditi0505) [inline pages/listing.html:156]: Same as above

File: styles/footer.css [hunk 4]
```diff
@@ -7,7 +7,13 @@
...
     background-color: var(--secondary-bg);
     border-top-left-radius: 15px;
-    border-top-right-radius: 15px;
+    border-width: 3px;
+    border-bottom: 0;
+    border-style: solid;
+    border-image:
+        linear-gradient(to left,
+            red,
+            rgba(0, 0, 0, 0))1 0 100%;
```
Comment (Aditi0505) [inline styles/footer.css]: You can use variable here too for providing color value.

**Review by Aditi0505 — APPROVED** [review]
Added few suggestions. Rest looks good to me.
Also, you can add the description, and screenshots for better visualization.

File: styles/footer.css [hunk 5]
```diff
@@ -7,7 +7,13 @@
     height: 40vh;
     background-color: var(--secondary-bg);
     border-top-left-radius: 15px;
```
Comment (sameep-s) [inline styles/footer.css:8]: you can use the border-radius short-hand for this.

File: pages/listing.html [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
```
Comment (sameep-s) [inline pages/listing.html:177]: you can add some value to alt tags.

**Review by sameep-s — APPROVED** [review]
left a few comments. take a look then you can merge.

**Pull request merged**